tests/serverpod_test_nonvector/serverpod_test_nonvector_server/lib/src/web/widgets/built_with_serverpod_page.dart

```diff
@@ -1,6 +1,6 @@
 import 'package:serverpod/serverpod.dart';
 
-class BuiltWithServerpodPage extends Widget {
+class BuiltWithServerpodPage extends Component {
```

Comment:
```suggestion
class BuiltWithServerpodPageComponent extends Component {
```

Only because we retain the suffix in the Google case.